life_expectancy/generate_samples.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+from . import DATA_DIR, FIXTURES_DIR
+from .cleaning import clean_data
+
+if __name__ == "__main__":
+
+    original_data = pd.read_csv(DATA_DIR / "eu_life_expectancy_raw.tsv", sep="\t")
+    generated_input = original_data.sample(n=10, random_state=0)
```

Comment:
It looks well-structured and functional, i  would add a docstring to clarify the function purpose.